public/support.html

```diff
@@ -94,29 +94,32 @@
     <link rel="stylesheet" href="css/header.css" />
     <link rel="stylesheet" href="css/styles.css?time=16258199860193" />
     <link rel="stylesheet" href=" https://cdn.devtron.ai/css/animations.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Barlow+Condensed:wght@300;600&display=swap" rel="stylesheet">
```

Comment:
Try to get this font family along with `Noto+Serif` from above

Reply:
Are they both same?